packages/react/.storybook/manager.tsx

```diff
@@ -32,3 +32,13 @@ addons.register('theme-synchronizer', () => {
         },
     });
 });
+
+addons.setConfig({
+    sidebar: {
+        filters: {
+            hidden: item => {
+                return !item.title.startsWith('Misc/');
```

Comment:
I guess you can remove `Misc` from `storySort` in `preview.tsx` now again as well.

Reply:
Thank you 🙏